src/main/java/bowling/utils/StringUtils.java

```diff
@@ -0,0 +1,20 @@
+package bowling.utils;
+
+import java.util.stream.Stream;
+
+public class StringUtils {
+    private final static String BLANK = " ";
+
+    private StringUtils() {}
+
+    public static boolean isEmpty(String value) {
+        return (value == null || "".equals(value.trim()));
+    }
+
+    public static String generateBlank(int length) {
+        if (length == 0) return "";
```

Comment:
```suggestion
    public static String generateBlank(int length) {
        if (length == 0) {
            return "";
        }
```

별도로 StringUtils 만들어주신점 좋습니다. 👍 
다만 indentation 깊이를 제가 확인하고 쉽도록, if 문과 for 문에는 brace 를 생략하지 말아주세요

Reply:
한줄 짜리 if 문에는 허용한다는 기준을 본 것 같은데 이건 컨벤션이 딱히 정해진게 없을까요?

Reply:
[Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.1-braces) 을 컨벤션으로 추천드립니다.
구글의 컨벤션에 따르면 한줄 짜리 if 문에도 brace 를 써달라고 되어 있습니다.